example-app/src/main/kotlin/org/eclipse/keyple/coppernic/ask/example/activity/MainActivity.kt

```diff
@@ -110,27 +99,32 @@ class MainActivity : AbstractExampleActivity(), PowerListener {
             }
 
             val askPlugin = SmartCardService.getInstance().registerPlugin(pluginFactory)
-            poReader = askPlugin.getReader(AndroidCoppernicAskContactlessReader.READER_NAME)
+            poReader = askPlugin.getReader(Cone2ContactlessReader.READER_NAME)
             (poReader as ObservableReader).addObserver(this@MainActivity)
 
             (poReader as ObservableReader).activateProtocol(
-                ContactlessCardCommonProtocols.ISO_14443_4.name,
-                ContactlessCardCommonProtocols.ISO_14443_4.name
+                ParagonSupportedContactlessProtocols.ISO_14443.name,
+                ParagonSupportedContactlessProtocols.ISO_14443.name
             )
 
-            //FIXME: Select the slot?
+            // FIXME: Select the slot?
             samReader = askPlugin.readers.toList().first().second
```

Comment:
It's a complex way to select a specific SAM slot.
I prefer to use name of the reader: "AndroidCoppernicAskContactReader_ONE" or "AndroidCoppernicAskContactReader_TWO"